build.yaml

```diff
@@ -662,6 +662,13 @@ common_cli:
   cleanup_old_jenkins_osx_yosemite_boxes: &cleanup_old_jenkins_osx_yosemite_boxes |
     admin/ci-tools/cleanup-vagrant-boxes clusterhq/jenkins-osx-yosemite
 
+  cleanup_cloud_resources: &cleanup_cloud_resources |
+    ${venv}/bin/python setup.py --version
+    # Cleanup leaked or stale cloud instances and volumes.
+    ${venv}/bin/python admin/cleanup_cloud_resources \
+    --config-file /tmp/acceptance.yaml \
+    -- 1>cleanup_cloud_resources.stdout 2>cleanup_cloud_resources.stderr
```

Comment:
Did you consider using tee here?

Using tee with stderr>stdout redirection would:

  * Allow us to see what the job is doing while it is running.
  * Keep interleaving of messages in the output files.

However, it would mean that didn't know whether a message went to which channel,
and perhaps lose the ability to process the output.